src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -5958,5 +5958,13 @@ class C
                 }
                 """);
         }
+
+        [Fact]
+        [WorkItem(70135, "https://github.com/dotnet/roslyn/issues/70135")]
```

Comment:
```suggestion
        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/70135")]
```

Reply:
done

Reply:
You could just accept the change in the UI. You merged attributes, but you should also omit issue id in `WorkItem` cnostructor since the url already includes it.

Reply:
Okay done next time i will do it from the UI